packages/venia-styleguide/src/pages/typography/content.yml

```diff
@@ -0,0 +1,2 @@
+---
+page_description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce quis euismod nisi. Morbi metus mauris, volutpat ac aliquet eget, laoreet vel tortor. Pellentesque commodo tellus."
```

Comment:
What do you think about using [MDX](https://github.com/mdx-js/mdx) for defining the page content/docs?

Reply:
Looks interesting! Probably a lot better than plain YAML or JSON.

Reply:
@jcalcaben Okay, I've added `@mdx-js/loader`, and I can confirm that it can successfully load and render `.md` files as MDX files.

I think we should probably restrict our usage of it to the sectional content for now. MDX files support some React basics and some JS primitives, but they're best for pure documents. We'll have to decide whether to inject components on a per-section basis or at the root with `MDXProvider`; I'm leaning toward the latter.

Reply:
I agree. Let's go with the latter and see what that experience is like.